cibuildwheel/windows.py

```diff
@@ -1,12 +1,18 @@
 from __future__ import print_function
-import os, tempfile, subprocess, shutil
+import os, tempfile, subprocess, shutil, sys
 from collections import namedtuple
 from glob import glob
 
+try:
+    from shlex import quote as shlex_quote
+except ImportError:
+    from pipes import quote as shlex_quote
+
 from .util import prepare_command, get_build_verbosity_extra_flags
 
 
 IS_RUNNING_ON_AZURE = os.path.exists('C:\\hostedtoolcache')
+IS_RUNNING_ON_TRAVIS = os.environ.get('TRAVIS_OS_NAME') == 'windows'
```

Comment:
Any reason not to use the constants from `utils.py`?

Reply:
Which utils? This from test?

Reply:
Oh, yes, that's code from the tests. Nevermind, my bad. Sorry!